handler/tokenizer_handler.py

```diff
@@ -58,4 +52,23 @@ def get_tokenizer(config_string: str) -> Language:
     if config_string not in __tokenizer_by_config_str:
         print(f"config string {config_string} not yet loaded", flush=True)
         init_tokenizer(config_string)
+        if get_config_value("is_managed"):
+            save_tokenizer_as_pickle(config_string)
+
     return __tokenizer_by_config_str[config_string]
+
+
+def save_tokenizer_as_pickle(config_string: str) -> None:
```

Comment:
not sure if nessecary but i'd somewhat expect some kind of "overwrite" flag that can be set in the endpoint to "refresh" the picked file if its corrupted (for whatever reason)

Reply:
added